draft-ietf-capport-architecture.xml

```diff
@@ -737,6 +737,19 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
      </list>
     </t>
     </section>
+    <section title="Handling of changed Portal URI">
+    <t>This section describes the possibility of changing Portal URI and how to handle
+      this. A different Portal URI could be returned in the following cases:</t>
+    <t>
+      <list style="symbols">
+        <t>If DHCP is used, a lease renewal/rebind may return a different Portal URI.</t>
```

Comment:
Should we be referring to the portal URI here, or is it rather the API URI?

Reply:
Consistency is king here.  "Captive Portal API" seems to be the full form, so you would say: 
```suggestion
        <t>If DHCP is used, a lease renewal/rebind may return a different Captive Portal API URI.</t>
```